crates/meroctl/src/cli/context/join.rs

```diff
@@ -54,6 +63,23 @@ impl JoinCommand {
 
         environment.output.write(&response);
 
+        if let Some(ref payload) = response.data {
+            if let Some(identity) = self.identity {
+                let context_id = payload.context_id;
+                let public_key = payload.member_public_key;
+
+                let res = create_alias(
+                    multiaddr,
+                    &config.identity,
+                    identity,
+                    Some(context_id),
+                    public_key,
+                )
+                .await?;
```

Comment:
if the context alias is specified, create a context alias
if the identity alias is specified, create an identity alias inside the context